mne_bids/utils.py

```diff
@@ -383,8 +385,12 @@ def _read_events(events_data, raw):
         The events file. If a string, a path to the events file. If an array,
         the MNE events array (shape n_events, 3). If None, events will be
         inferred from the stim channel using `find_events`.
+    event_id : dict
+        The event_id provided in write_raw_bids.
```

Comment:
This description is not very telling, I would change this to what we have here:

https://github.com/mne-tools/mne-bids/blob/187c5619481aa8c8853fb2fb9272a95bb0a6bcae/mne_bids/mne_bids.py#L173-L175

perhaps: `The event_id dict as provided in write_raw_bids, mapping a brief description key to an event id (value).`